docs/reference/content/guides/annotations.md

```diff
@@ -66,6 +76,29 @@ In contrast to `@Reference` where a nested Java reference ends up as a separate
 in the document of the parent object.  This annotation can be applied to the class of the embedded type or on the field holding the
 embedded instance.
 
+### Validation
+[`@Validation`]({{< apiref "org/mongodb/morphia/annotations/Validation" >}}) allows for the definition of a 
+[document validation]({{< docsref "core/document-validation/" >}}) schema to applied to all writes to MongoDB.  Validation rules are 
+specified on a per-collection basis using any query operators, with the exception of `$near`, `$nearSphere`, `$text`, and `$where`.  This
+validation definition is done using the MongoDB query syntax as shown here:
+
+```java
+@Validation("{ number : { $gt : 10 } }")
+public class SomeEntity {
+    ...
+    private int number;
+    ...
+}
+```
+
+This validation is applied to every insert and update in the mapped collection.  In some cases, it's necessary to bypass this validation. To
+ this end, the various operations on [`Datastore`]({{< apiref "org/mongodb/morphia/Datastore" >}}) and 
+[`AdvancedDatastore`]({{< apiref "org/mongodb/morphia/AdvancedDatastore" >}}) take Options classes.  There are varying options to be set on 
```

Comment:
Should be just collapse the three sentences to:
Various [`Datastore`] and [`AdvancedDatastore`] operations can bypass this validation via their Options classes. For these operations, specify the `bypassDocumentValidation` option  to disable document validation for a specific operation.